spec/defines/vhost_spec.rb

```diff
@@ -192,6 +192,7 @@
           'php_admin_values'            => ['true', 'false'],
           'no_proxy_uris'               => '/foo',
           'proxy_preserve_host'         => true,
+          'proxy_error_override'        => true,
```

Comment:
I was unter the impression, that the context was just to "set everything!" and check whether the manifest compiles. What tests do you think are necessary?